storage/google/cloud/storage/bucket.py

```diff
@@ -943,8 +1115,46 @@ def lifecycle_rules(self, rules):
         :type entries: list of dictionaries
         :param entries: A sequence of mappings describing each lifecycle rule.
         """
+        rules = [dict(rule) for rule in rules]  # Convert helpers if needed
         self._patch_property('lifecycle', {'rule': rules})
 
+    def clear_lifecyle_rules(self):
+        """Set lifestyle rules configured for this bucket.
+
+        See https://cloud.google.com/storage/docs/lifecycle and
+             https://cloud.google.com/storage/docs/json_api/v1/buckets
+        """
+        self.lifecycle_rules = []
+
+    def add_lifecycle_delete_rule(self, **kw):
+        """Add a "delete" rule to lifestyle rules configured for this bucket.
```

Comment:
Could you add an example snippet in the documentation:

```python
from google.cloud import storage

storage_client = storage.Client()
bucket = storage_client.get_bucket("bucket-name")
bucket.add_lifecycle_delete_rule(age=2)
bucket.patch()
```

Reply:
d7674b1